src/Directory.Build.props

```
@@ -90,6 +94,11 @@
 		<TargetPlatformMinVersion>10.14</TargetPlatformMinVersion>
 	</PropertyGroup>
 
+	<ItemGroup>
+		<None Include="build\Package.targets" Pack="true" PackagePath="buildTransitive\$(PackageId).targets" Condition="exists('build') and !$(PackageId.Contains('WinUI'))"/>
+		<None Include="build\Package.targets" Pack="true" PackagePath="buildTransitive\$(PackageId).targets" Condition="exists('build') and $(PackageId.Contains('WinUI'))"/>
```

Sounds like the condition `$(PackageId.Contains('WinUI'))` is useless :)

```suggestion
		<None Include="build\Package.targets" Pack="true" PackagePath="buildTransitive\$(PackageId).targets" Condition="exists('build')"/>
```